StortSpelProjekt/Game/src/Gamefiles/EnemyFactory.cpp

```diff
@@ -30,7 +30,7 @@ void EnemyFactory::SetScene(Scene* scene)
 	m_pScene = scene;
 }
 
-Entity* EnemyFactory::AddEnemy(const std::string& entityName, Model* model, int hp, float3 pos, const std::wstring& sound3D, unsigned int compFlags, unsigned int aiFlags, float scale, float3 rot, const std::string& aiTarget, float aiDetectionRadius, float aiAttackingDistance, float aiAttackInterval)
+Entity* EnemyFactory::AddEnemy(const std::string& entityName, Model* model, int hp, float3 pos, const std::wstring& sound3D, unsigned int compFlags, unsigned int aiFlags, float scale, float3 rot, const std::string& aiTarget, float aiDetectionRadius, float aiAttackingDistance, float aiAttackInterval, float aiMeleeAttackDmg)
```

Comment:
When merging in develop, make sure this addition is also added to the DefineEnemy() function